src/lib/dnssd/Constants.h

```diff
@@ -64,11 +64,11 @@ constexpr size_t kSubTypeTotalLength    = chip::Sum(SUBTYPES);
  * Matter commissionable/commissioner node service constants.
  */
 
-namespace Commissionable {
+namespace Commission {
```

Comment:
I assume the point. of this is for this namespace to be able to encompass both of commissioner discovery and commissionable node discovery.

Perhaps it is a personal preference, but this feels slightly more natural:

```suggestion
namespace Commissioning {
```

Reply:
Commission has the advantage to be shorter .... lets keep it this way